src/vivarium_gates_mncnh/data/lbwsg_paf.yaml

```diff
@@ -3,15 +3,16 @@ components:
         components:
             - PAFResultsStratifier()
             - EvenlyDistributedPopulation()
+            - LBWSGMortality()

```

Comment:
How are we able to include this component in the PAF sim, when it seems like it would itself require a PAF?

Reply:
We only have ACMR here and define the PAF as 0 in the component itself.

Reply:
If we define the PAF to be zero, won't more simulants die in the ENN period than should, and mess up our LNN exposure distribution?

Since early neonates will get LBWSG RRs (>= 1) applied _on top of_ GBD's ACMR, we would end up with an ACMR higher than GBD, no?